app/src/main/java/org/wikipedia/theme/ThemeFittingRoomActivity.kt

```diff
@@ -0,0 +1,44 @@
+package org.wikipedia.theme
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.core.content.ContextCompat
+import org.wikipedia.Constants
+import org.wikipedia.activity.SingleFragmentActivity
+import org.wikipedia.page.ExclusiveBottomSheetPresenter
+
+class ThemeFittingRoomActivity : SingleFragmentActivity<ThemeFittingRoomFragment>(), ThemeChooserDialog.Callback {
+    private var themeChooserDialog: ThemeChooserDialog? = null
+    private val bottomSheetPresenter = ExclusiveBottomSheetPresenter()
+    override fun onCreate(savedInstanceState: Bundle?) {
```

Comment:
Sorry I didn't notice this but is it possible to add an empty line between `onCreate` and the variables?

Reply:
Done @cooltey